package.json

```diff
@@ -12,6 +12,10 @@
     "Google Maps",
     "map"
   ],
+  "scripts": {
+    "lint": "eslint src --ext=js --format=unix",
```

Comment:
You are correct.
The format switch is to make file path detection work in the console.